src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -211,13 +211,20 @@ function toggleCollapse(e, pans) {
     var panels = pans || $('div.panel-collapse');
 
     if ($(e).html().trim().startsWith(expand)) {
-        var i = 0;
-        for (var idx = 0; idx < panels.length; idx++) {
-            if ($(panels[idx]).attr('class').indexOf('in') === -1) {
-                setTimeout(showSingleCollapse, 50 * i, panels[idx]);
-                i++;
+        $('.ajax-response-auto').click();
+        var $autoLoad = $('.ajax_auto').click();
+        var isAutoLoadingTriggered = $autoLoad.length !== 0;
+
+        if (!isAutoLoadingTriggered) {
```

Comment:
What's this whole block doing? Either add a comment to explain or extract into a function with an explanatory name.